Core/Frameworks/Baikal/Framework.php

```diff
@@ -56,22 +54,18 @@ static function bootstrap() {
         ) {
             self::installTool();
         } else {
-            require_once(PROJECT_PATH_SPECIFIC . "config.php");
-            require_once(PROJECT_PATH_SPECIFIC . "config.system.php");
+            require_once PROJECT_PATH_SPECIFIC . "config.php";
```

Comment:
There are a few of these, remove the `()` for `require_once`. I couldn't quickly find which fixer to disable. But probably this is only a small change anyway, so let it do it.

Reply:
I think this will cause many merge conflicts with #835. Your style changes were done automatically, right? Then I would vote to wait for #835 and run the cs fixer again after merging that one.

Reply:
I rebased just now.
The changes in this PR are not super-important, just a refactoring of the cs-fixer in use. So it will be easy to wait and rebase/reapply the code-style after that.

@ByteHamster what is the state of #835 ?

It does not seem to be moving since a few months ago.

Reply:
> I think this will cause many merge conflicts with #835. Your style changes were done automatically, right? Then I would vote to wait for #835 and run the cs fixer again after merging that one.

#835 is merged now. Ready to merge this one if you could rebase again (take your time)